app/middleware/locals.js

```diff
@@ -0,0 +1,11 @@
+module.exports = (config) =>
+  (req, res, next) => {
+    /* eslint-disable no-param-reassign */
+    res.locals.GOOGLE_ANALYTICS_TRACKING_ID = config.googleAnalyticsId;
+    res.locals.WEBTRENDS_TRACKING_ID = config.webtrendsId;
+    res.locals.HOTJAR_TRACKING_ID = config.hotjarId;
+    res.locals.FONT_CDN_HOST = config.fontCdnHost;
```

Comment:
Good, spot toyed with adding it but decided it was out of scope and left in in error.